hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/AutoCreationLogic.java

```diff
@@ -240,7 +259,11 @@ public Pair<ResponseCodeEnum, Long> create(
         // If the transaction fails, we will get an opportunity to unlink this alias in
         // reclaimPendingAliases()
```

Comment:
Should the memo be `LAZY_MEMO` in the child record on lazy creates?

Reply:
Yeah, I think it should. I will change it there as well.